packages/connection-form/src/components/advanced-options-tabs/tls-ssl-tab/tls-certificate-authority.tsx

```diff
@@ -26,20 +39,44 @@ function TLSCertificateAuthority({
     <FormFieldContainer className={caFieldsContainer}>
       <FileInput
         description={'Learn More'}
-        disabled={disabled}
+        disabled={disabled || useSystemCA}
         id="tlsCAFile"
         dataTestId="tlsCAFile-input"
         label="Certificate Authority (.pem)"
         link={
           'https://docs.mongodb.com/manual/reference/connection-string/#mongodb-urioption-urioption.tlsCAFile'
         }
         onChange={(files: string[] | null) => {
-          updateCAFile(files && files.length > 0 ? files[0] : null);
+          updateCAFile(files && files.length > 0 ? files[0] : null, false);
         }}
         showFileOnNewLine
         values={caFile ? [caFile] : undefined}
         optional
       />
+      <Checkbox
+        onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+          updateCAFile(null, event.target.checked);
+        }}
+        data-testid="useSystemCA-input"
+        id="useSystemCA-input"
+        label={
+          <>
+            <Label htmlFor="useSystemCA-input">
+              Use System Certificate Authority
+            </Label>
+            <Description
+              className={cx(checkboxDescriptionStyles, {
+                [disabledCheckboxDescriptionStyles]: disabled,
+              })}
+            >
+              Use the operating system’s Certificate Authority store.
+            </Description>
+          </>
+        }
+        disabled={disabled}
+        checked={useSystemCA}
+        bold={false}
```

Comment:
V small suggestion, looks like `bold={false}`[ is the default](https://www.mongodb.design/component/checkbox/documentation/), so we can probably leave that out
```suggestion
```
I think it exists on one or two of the checkboxes in the connect form which we can probably remove it from too, probably left over from when we were passing `label` as a string.

Reply:
Yeah, I just copied it from the other TLS checkboxes but removing it doesn’t appear to make a difference, done :+1: